app/src/main/java/io/homeassistant/companion/android/settings/widgets/views/ManageWidgetsView.kt

```diff
@@ -110,7 +114,7 @@ fun ManageWidgetsView(
         ) {
             if (viewModel.buttonWidgetList.value.isEmpty() && viewModel.staticWidgetList.value.isEmpty() &&
                 viewModel.mediaWidgetList.value.isEmpty() && viewModel.templateWidgetList.value.isEmpty() &&
-                viewModel.cameraWidgetList.value.isEmpty()
+                viewModel.templateWidgetList.value.isEmpty() && viewModel.graphWidgetList.value.isEmpty()

```

Comment:
Looks like you may have mixed up `cameraWidgetList` with `templateWidgetList` here :)